src/core_modules/capture-core/components/WidgetStagesAndEvents/types/common.types.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+type DataValue = {
```

Comment:
Hey,

Do you think it makes sense to reuse `type ApiDataValue` and `type ApiTEIEvent` from `getEnrollmentEvents.js` instead of creating new types? 
Thanks!

Reply:
@simonadomnisoru  thanks for the suggestion, I didn't know this type exist :) updated the branch